### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by windayski**
fix broken links

### CODE REVIEW

File: book/src/snapshot-restore-feature.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Unlike the core Kubernetes Persistent Volume objects, these Snapshot objects are
 
 The CRDs are [automatically deployed](https://github.com/kubernetes-csi/external-snapshotter/blob/master/cmd/csi-snapshotter/create_crd.go#L29) by the CSI [external-snapshotter](external-snapshotter.md) sidecar.
 
-The schema definition for the custom resources (CRs) can be found here: [https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go](https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go)
+The schema definition for the custom resources (CRs) can be found here: [https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1beta1/types.go](https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go)
```
Comment (ydcool) [inline book/src/snapshot-restore-feature.md]: the real link should be updated  as well
```diff
- [https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go](https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1alpha1/types.go)
+ [https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1beta1/types.go](https://github.com/kubernetes-csi/external-snapshotter/blob/master/pkg/apis/volumesnapshot/v1beta1/types.go)
```

**Comment by windayski** [comment]
@ydcool Thanks. The real link is updated.

File: book/src/external-snapshotter.md [hunk 2]
```diff
@@ -34,4 +34,4 @@ For detailed information (binary parameters, RBAC rules, etc.), see [https://git
 
 The CSI `external-snapshotter` is deployed as a controller. See [deployment section](deploying.md) for more details.
 
-For an example deployment, see [this example](https://github.com/kubernetes-csi/external-snapshotter/tree/master/deploy/kubernetes/setup-csi-snapshotter.yaml) which deploys `external-snapshotter` and `external-provisioner` with the Hostpath CSI driver.
+For an example deployment, see [this example](https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml) which deploys `external-snapshotter` and `external-provisioner` with the Hostpath CSI driver.
```
Comment (msau42) [inline book/src/external-snapshotter.md]: The entire snapshot pages are being reworked in https://github.com/kubernetes-csi/docs/pull/260.

Can you hold off on the snapshot page changes, and just include the troubleshooting one?

**Comment by xing-yang** [comment]
FYI - there's a PR being reviewed currently to fix broken snapshot links.

**Pull request closed without merging**